ippserver/ppd.py

```diff
@@ -2,12 +2,24 @@
 from __future__ import absolute_import
 from __future__ import print_function
 
+from .server import local_file_location
+
 
 class PPD(object):
     def text(self):
         raise NotImplementedError()
 
 
+class FilePPD(PPD):
```

Comment:
We should probably make all the other PPD classes into normal files too - setup.py handles [data_files](https://docs.python.org/2/distutils/setupscript.html#installing-additional-files), so this shouldn't be _too_ bad. But I won't force you to do that as part of this PR!